### PR TITLE
LoadCssAsync html format fixed for critical css 

### PR DESCRIPTION
**Pull request by srsathish92**
### Description (*)
LoadCssAsync html format fixed

### Fixed Issues (if relevant)
1. magento/magento2#26760: Validate html error when enable critical css

### Manual testing scenarios (*)

1. ...
2. ...

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Review by ihor-sviziev — CHANGES_REQUESTED** [review]
@srsathish92 
Your changes looks good to me.
Could you cover your changes with some tests in order to prevent appearing this issue in future?

**Comment by srsathish92** [comment]
@ihor-sviziev I have covered unit test for `AsyncCssPlugin.php` & updated. can you please review now.

File: app/code/Magento/Theme/Controller/Result/AsyncCssPlugin.php [hunk 1]
```diff
@@ -16,7 +16,7 @@
  */
 class AsyncCssPlugin
 {
-    private const XML_PATH_USE_CSS_CRITICAL_PATH = 'dev/css/use_css_critical_path';
+    const XML_PATH_USE_CSS_CRITICAL_PATH = 'dev/css/use_css_critical_path';
```
Comment (ihor-sviziev) [inline app/code/Magento/Theme/Controller/Result/AsyncCssPlugin.php]: Please revert changing constant visibility. As I understood - you needed it in tests, so please just create own const inside test class and use it there

**Review by ihor-sviziev — CHANGES_REQUESTED** [review]
Hi,
Changes looks really good for me, only one small adjustment still needed.
Failing integration tests looks not related to this PR

File: app/code/Magento/Theme/Controller/Result/AsyncCssPlugin.php (hunk 1 above)
Comment (srsathish92) [reply app/code/Magento/Theme/Controller/Result/AsyncCssPlugin.php]: Yeah, sometimes failing integration tests like this, as requested I have reverted the constant change & added new stub in test class. can you please review now.

Comment (srsathish92) [reply app/code/Magento/Theme/Controller/Result/AsyncCssPlugin.php]: Thanks for the feedback

**Review by ihor-sviziev — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @ihor-sviziev, thank you for the review.
[ENGCOM-6849](https://jira.corp.magento.com/browse/ENGCOM-6849) has been created to process this Pull Request

**Comment by engcom-Delta** [comment]
:heavy_check_mark: QA passed

**Review by sudheers-kensium — APPROVED** [review]

**Comment by engcom-Echo** [comment]
Failed functional test not related to the changes in this PR test fails in 2.4-develop with MSI modules.

**Pull request merged**